src/file_utils.cc

```diff
@@ -142,8 +143,6 @@ int _gzdecompress_file(const char* existingFilePath, const char* newFilePath)
 
 void fileCopy(const char* existingFilePath, const char* newFilePath)
 {
-    std::ifstream source { existingFilePath, std::ios::binary };
-    std::ofstream destination { newFilePath, std::ios::binary };
-
-    destination << source.rdbuf();
+    std::error_code ec;
+    std::filesystem::copy_file(std::filesystem::path(existingFilePath), std::filesystem::path(newFilePath), ec);
```

Comment:
Not sure if it is needed but the original CopyFileA function overwrote the destination file but the std::filesystem::copy_file will fail without the [std::filesystem::copy_options::overwrite_existing](https://en.cppreference.com/w/cpp/filesystem/copy_options) option